templates/web/base/tokens/confirm_problem.html

```diff
@@ -7,17 +7,30 @@
   [% IF c.cobrand.is_council %]
     [% IF c.cobrand.owns_problem( report ) %]
         <h2>Your issue is on its way to the council.</h2>
+        <p>Your reference for this report is [% report.id %], please quote it in any enquiries.</p>
+        <p><a class="green-btn" href="[% c.cobrand.base_url_for_report( report ) %][% report.url %]">View report</a></p>
     [% ELSE %]
-        <h2>Your issue is on its way to [% report.body %].</h2>
-        <p>Note that [% c.cobrand.council_name %] is not responsible for this type of problem.</p>
-    [% END %]
-    <p>Your reference for this report is [% report.id %], please quote it in any enquiries.</p>
+        <h2>Thank you for your report.</h2>
+        <p>
+            We don't handle this type of problem, so have passed it on to:
+        </p>
+        <p>
+            <b>[% report.body %]</b>
+        </p>
+        <p>
+            You can follow this problem on the main FixMyStreet website.
+        </p>
+        <p>
+            <a class="green-btn" href="[% c.cobrand.base_url_for_report( report ) %][% report.url %]">Go to FixMyStreet</a>
+        </p>
+     [% END %]
 
   [% ELSE %]
     <h2>[% loc('Thank you for reporting this issue!') %]</h2>
     [% IF report.bodies_str %]
       <p>[% loc('It’s on its way to the council right now.') %]</p>
     [% END %]
+    <p><a class="green-btn" href="[% c.cobrand.base_url_for_report( report ) %][% report.url %]">View report</a></p>
```

Comment:
Will want to check this with @zarino who presumably deliberately made the title the link here, and might want this displayed a bit differently. This also needs i18n, and also the styling is wrong with fixmystreet.com styling – blue link, green background.